framework/DataObjects/DataSet.py

```diff
@@ -125,11 +124,11 @@ def addExpectedMeta(self,keys):
     self._metavars.extend(keys)
     self._orderedVars.extend(keys)
 
-  def addMeta(self,tag,xmlDict):
+  def addMeta(self, tag, xmlDict = None, node = None):
```

Comment:
For what it is worth, in PEP 8 this should be: ```def addMeta(self, tag, xmlDict=None, node=None):```

Reply:
Oh, thanks! I missed the exception for function keywords.